main.c

```diff
@@ -432,6 +432,43 @@ init_headless(struct vkcube *vc)
                  NULL,
                  &b->image);
 
+   VkFormatProperties2 formatProperties = {
```

Comment:
In the vkcube code, the naming convention is `snake_case` not `camelCase`. Either `format_properties` or `format_props` is good.